### PR TITLE
[ZH] Update `Account`

### PR DESCRIPTION
**Pull request by Coin-233**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by TPGPL** [comment]
the translation is still missing some content from the English article

File: wiki/Help_centre/Account/zh.md [hunk 1]
```diff
@@ -225,6 +223,18 @@ outdated_since: ef784b9104fbcc3a66f3bf503d97147aa7b26345
...
 
+### 我不想继续玩了，但我不想我的账号被删除！ {#account-lock}
+
+任何用户都可以通过向[accounts@ppy.sh](mailto:accounts@ppy.sh)发送电子邮件来请求**锁定**他们的账号。该账号的密码将被打乱，其电子邮件地址将被清空，从而将用户锁定在账号之外。用这种方式锁定账号后，有一周的冷却时间，然后用户才能请求将其解锁，以防止滥用。
+
+锁定的账号将保持公开可见，而不会更改其个人资料中的可用信息。
+
+要解锁账号，请使用请求锁定账号中相同的的电子邮件地址向[accounts@ppy.sh](mailto:accounts@ppy.sh)发送电子邮件，并表达您解锁账号的意图。
+
+此功能不可滥用。对于那些不想离开osu的玩家来说，锁定账号应该是最后的手段之一！永远，但**需要**专门的时间远离 osu！出于正当理由。
```
Comment (YumeMuzi) [inline wiki/Help_centre/Account/zh.md:234]:
```suggestion
不要滥用此功能。锁定帐户应该是那些不想永远地离开 osu!，但出于正当理由，**需要**专心做事从而离开 osu! 的玩家的最后手段！
```

File: wiki/Help_centre/Account/zh.md [hunk 2]
```diff
@@ -225,6 +223,18 @@ outdated_since: ef784b9104fbcc3a66f3bf503d97147aa7b26345
 
 是的，任何用户可以发邮件至 [privacy@ppy.sh](mailto:privacy@ppy.sh) 请求删除帐号。请记住，删除您的帐户是**永久操作**，这意味着它无法恢复或撤销。删除你的帐号不意味着你可以创建一个新账户，拥有多个帐号违反 [规章制度](/wiki/Rules#社区规定)。这意味着你将无法再使用 osu! 的在线功能。
 
+### 我不想继续玩了，但我不想我的账号被删除！ {#account-lock}
+
+任何用户都可以通过向[accounts@ppy.sh](mailto:accounts@ppy.sh)发送电子邮件来请求**锁定**他们的账号。该账号的密码将被打乱，其电子邮件地址将被清空，从而将用户锁定在账号之外。用这种方式锁定账号后，有一周的冷却时间，然后用户才能请求将其解锁，以防止滥用。
```
Comment (YumeMuzi) [inline wiki/Help_centre/Account/zh.md:228]:
```suggestion
任何用户都可以通过向 [accounts@ppy.sh](mailto:accounts@ppy.sh) 发送邮件来请求**锁定**他们的账号。锁定后，将打乱账号的密码，并清空邮箱地址，从而将用户隔离在账号之外。为防止滥用，锁定账号后有一周的冷却时间，之后用户才能请求解锁账号。
```

File: wiki/Help_centre/Account/zh.md [hunk 3]
```diff
@@ -225,6 +223,18 @@ outdated_since: ef784b9104fbcc3a66f3bf503d97147aa7b26345
...
+
+锁定的账号将保持公开可见，而不会更改其个人资料中的可用信息。
```
Comment (YumeMuzi) [inline wiki/Help_centre/Account/zh.md:230]:
```suggestion
锁定的账号将保持公开可见，账号的个人资料不会更改。
```

File: wiki/Help_centre/Account/zh.md [hunk 4]
```diff
@@ -225,6 +223,18 @@ outdated_since: ef784b9104fbcc3a66f3bf503d97147aa7b26345
...
+
+要解锁账号，请使用请求锁定账号中相同的的电子邮件地址向[accounts@ppy.sh](mailto:accounts@ppy.sh)发送电子邮件，并表达您解锁账号的意图。
+
+此功能不可滥用。对于那些不想离开osu的玩家来说，锁定账号应该是最后的手段之一！永远，但**需要**专门的时间远离 osu！出于正当理由。
+
+您可以通过将您的电子邮件和密码更改为您永远不会记住的随机单词或字母并退出账号来将自己锁定在您的帐户之外。这与锁定基本上是相同的过程。准备好重返社区后，请发送电子邮件至[accounts@ppy.sh](mailto:accounts@ppy.sh)请求重置密码。确保您使用您的帐户之前使用的电子邮件地址。
```
Comment (YumeMuzi) [inline wiki/Help_centre/Account/zh.md:236]:
```suggestion
您可以将您的邮箱和密码更改为永远不会记住的随机单词或字母，并退出账号，来将自己隔离在您的帐户之外。这与锁定账号基本相同。当你准备好重返社区后，请使用之前用于锁定账号的邮箱，发送电子邮件至[accounts@ppy.sh](mailto:accounts@ppy.sh)请求重置账号密码。
```

File: wiki/Help_centre/Account/zh.md [hunk 5]
```diff
@@ -225,6 +223,18 @@ outdated_since: ef784b9104fbcc3a66f3bf503d97147aa7b26345
 
 是的，任何用户可以发邮件至 [privacy@ppy.sh](mailto:privacy@ppy.sh) 请求删除帐号。请记住，删除您的帐户是**永久操作**，这意味着它无法恢复或撤销。删除你的帐号不意味着你可以创建一个新账户，拥有多个帐号违反 [规章制度](/wiki/Rules#社区规定)。这意味着你将无法再使用 osu! 的在线功能。
 
+### 我不想继续玩了，但我不想我的账号被删除！ {#account-lock}
+
+任何用户都可以通过向[accounts@ppy.sh](mailto:accounts@ppy.sh)发送电子邮件来请求**锁定**他们的账号。该账号的密码将被打乱，其电子邮件地址将被清空，从而将用户锁定在账号之外。用这种方式锁定账号后，有一周的冷却时间，然后用户才能请求将其解锁，以防止滥用。
+
+锁定的账号将保持公开可见，而不会更改其个人资料中的可用信息。
+
+要解锁账号，请使用请求锁定账号中相同的的电子邮件地址向[accounts@ppy.sh](mailto:accounts@ppy.sh)发送电子邮件，并表达您解锁账号的意图。
```
Comment (YumeMuzi) [inline wiki/Help_centre/Account/zh.md:232]:
```suggestion
要解锁账号，请使用请求锁定账号时相同的邮箱向 [accounts@ppy.sh](mailto:accounts@ppy.sh) 发送邮件，并表露您要解锁账号的意图。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
Notice, you should **NEVER** directly use the results in the translation software.

注意，汉化时请永远**不要**使用机翻。

File: wiki/Help_centre/Account/zh.md (hunk 1 above)
Comment (Moutreak) [reply wiki/Help_centre/Account/zh.md:234]:
```suggestion
不要滥用此功能。锁定账号应当是以下玩家的一种最终手段：不想永远离开osu!, 但有正当理由表明其需要刻意远离osu!一段时间。
```

File: wiki/Help_centre/Account/zh.md (hunk 3 above)
Comment (Moutreak) [inline wiki/Help_centre/Account/zh.md:230]:
```suggestion
锁定的账号将保持公开可见，其个人资料上可见的信息不会被更改。
```

File: wiki/Help_centre/Account/zh.md (hunk 5 above)
Comment (Moutreak) [inline wiki/Help_centre/Account/zh.md:232]:
```suggestion
要解锁账号，请向 [accounts@ppy.sh](mailto:accounts@ppy.sh) 发送邮件并陈述解锁意图，使用的邮箱地址必须和请求锁定邮件时的相同。
```

File: wiki/Help_centre/Account/zh.md (hunk 4 above)
Comment (Moutreak) [inline wiki/Help_centre/Account/zh.md:236]:
```suggestion
如果不想访问自己的账号，你可以把邮箱地址和密码改成你永远记不住的随机字符串，然后登出账号。这和锁定基本上是相同的过程。当你准备好重返社区时，用之前的邮箱地址发送邮件至 [accounts@ppy.sh](mailto:accounts@ppy.sh) 请求重置密码即可。
```

**Review by Moutreak — CHANGES_REQUESTED** [review]
Improved translation by myself.

**Pull request closed without merging**